### PR TITLE
Fix #808 - relative symlinks

### LINKED ISSUE #808: Absolute paths in bin symlinks
`NIMBLE_DIR` can not be moved/renamed because the symlinks in `NIMBLE_DIR/bin` are absolute. Moving NIMBLE_DIR around is handy in CI environments.

### PR DESCRIPTION
**Pull request by genotrance**
cc @yglukhov

### CODE REVIEW

File: src/nimblepkg/packageinstaller.nim [hunk 1]
```diff
@@ -62,14 +64,14 @@ proc setupBinSymlink*(symlinkDest, symlinkFilename: string,
       if fixChcp:
         contents.add "chcp 65001 > nul && "
       else: contents.add "chcp 65001 > nul\n@"
-    contents.add "\"" & symlinkDest & "\" %*\n"
+    contents.add "\"%~dp0\\" & symlinkDestRel & "\" %*\n"
```
Comment (dom96) [inline src/nimblepkg/packageinstaller.nim:67]: damn, this is some magic. You might want to consider using raw string literals here.

**Review by dom96 — CHANGES_REQUESTED** [review]
This seems good, but it needs a test plan. Can you show how you've tested this change together with the results?

**Comment by genotrance** [comment]
> This seems good, but it needs a test plan. Can you show how you've tested this change together with the results?

I manually verified that the shims are created with relative paths on Win/Lin and the commands still work. The CI builds and runs many binaries so I figured it got tested as well.

File: src/nimblepkg/packageinstaller.nim (hunk 1 above)
Comment (genotrance) [reply src/nimblepkg/packageinstaller.nim:67]: Raw strings don't help since the string contains `"` and `\n`. Doesn't look any prettier.

**Pull request merged**